indico/web/forms/fields/simple.py

```diff
@@ -147,5 +147,13 @@ def _value(self):
         return self.text_value
 
 
+class IndicoEmailRecipientsField(IndicoStaticTextField):
```

Comment:
I'd change this inheritance (probably cleaner to keep these fields separate) and pass a list of emails to it.